### PR TITLE
Work item updates

### PR DESCRIPTION
**Pull request by aboba**

### CODE REVIEW

File: webrtc-charter.html [hunk 1]
```diff
@@ -233,6 +233,12 @@ <h3>
             <dd>JavaScript APIs that let a Web application manage how audio is rendered on the user audio output devices</dd>
             <dt><a href="http://w3c.github.io/mediacapture-screen-share/">Screen Capture</a></dt>
             <dd>An extension to the Media Capture and Streams API to use a user's display, or parts thereof, as the source of a MediaStream.</dd>
+	    <dt><a href="http://w3c.github.io/webrtc-quic/">QUIC in WebRTC</a></dt>
+            <dd>An extension to the WebRTC 1.0 API to enable use of peer-to-peer QUIC streams for data exchange.</dd>
+            <dt><a href="http://w3c.github.io/webrtc-ice/">IceTransport extensions</a></dt>
+            <dd>An extension to the WebRTC 1.0 API to enable construction of RTCIceTransport objects.</dd>
+            <dt><a href="http://w3c.github.io/webrtc-qos/">QoS extensions</a></dt>
```
Comment (alvestrand) [inline webrtc-charter.html]: should be webrtc-dscp-exp, and called "Experimental controls for DSCP markings"

File: webrtc-charter.html [hunk 2]
```diff
@@ -233,6 +233,12 @@ <h3>
             <dd>JavaScript APIs that let a Web application manage how audio is rendered on the user audio output devices</dd>
             <dt><a href="http://w3c.github.io/mediacapture-screen-share/">Screen Capture</a></dt>
             <dd>An extension to the Media Capture and Streams API to use a user's display, or parts thereof, as the source of a MediaStream.</dd>
```
Comment (alvestrand) [inline webrtc-charter.html:235]: Can add links to these 3 too, for consistency

**Review by stefhak — APPROVED** [review]

File: webrtc-charter.html [hunk 3]
```diff
@@ -233,6 +233,12 @@ <h3>
...
+	    <dt><a href="https://github.com/w3c/webrtc-quic">QUIC in WebRTC</a></dt>
+            <dd>An extension to the WebRTC 1.0 API to enable use of peer-to-peer QUIC streams for data exchange.</dd>
+            <dt><a href="https://github.com/w3c/webrtc-ice/">IceTransport extensions</a></dt>
+            <dd>An extension to the WebRTC 1.0 API to enable construction of RTCIceTransport objects.</dd>
+            <dt><a href="https://github.com/w3c/webrtc-dscp-exp/">Experimental QoS extensions</a></dt>
+            <dd>An experimental extension to the WebRTC 1.0 API to enable QoS marking.</dd>
```
Comment (dontcallmedom) [inline webrtc-charter.html]: I don't think we should call it experimental in the charter - while it is experimental at the moment, presumably we would want to make it "real" during the charter.

**Comment by dontcallmedom** [comment]
I noted a minor nit probably worth fixing; I think we also need to update the description of WebRTC NV (either in this PR or separately):

> The activities in the ORTC (Object Real-time Communications) Community Group indicate that there is interest in additional APIs to provide more direct control over WebRTC than what the PeerConnection API offers.
> 
> In recognition of this interest, the Working Group will, once WebRTC 1.0: Real-time Communication Between Browsers reaches Candidate Recommendation, start work on a new set of object-oriented APIs for real-time communication.
> 
> In developing these new APIs, the Working Group will adhere to the following principles:
> 
> *    Direct control: The new APIs are intended to provide direct control over the details of real-time communication, where the application can directly specify how information should be transmitted, without any built-in negotiation semantics.
> *    Standalone operation: The new APIs will be complete enough to allow applications to write solely to the new APIs to complete common tasks.
> *    Backwards-compatibility: The new APIs will extend the WebRTC 1.0 APIs, rather than replace them. Applications that use the PeerConnection API will continue to function, unless there is a clear and compelling reason to deprecate specific 1.0 functionality.
> *    Feature independence: Features may be introduced in the new APIs that are not available when using the PeerConnection API.
> 
> The Working Group will take the work done by the ORTC Community Group as a source of input, and when contemplating similar APIs in the Working Group, make efforts to align with the ORTC CG on API methodologies and nomenclature. This may include scheduled design meetings with relevant WG and CG stakeholders to foster convergence of the APIs.

File: webrtc-charter.html (hunk 3 above)
Comment (alvestrand) [reply webrtc-charter.html]: We'd want to run an experiment to figure out whether or not this is a control surface worth having?

**Pull request merged**